dev/CsvBulkLoader.php

```diff
@@ -47,8 +56,115 @@ public function preview($filepath) {
 	/**
 	 * @param string $filepath
 	 * @param boolean $preview
+	 *
+	 * @return null|BulkLoader_Result
 	 */
 	protected function processAll($filepath, $preview = false) {
+		$files = $this->splitFile($filepath);
+
+		$result = null;
+		$last = null;
+
+		try {
+			foreach ($files as $file) {
+				$last = $file;
+
+				$next = $this->processChunk($file, false);
+
+				if ($result instanceof BulkLoader_Result) {
+					$result->merge($next);
+				} else {
+					$result = $next;
+				}
+
+				@unlink($file);
```

Comment:
Hmm, `@` error suppression? As a dev, I'd want to know if deleting this file failed - it might cause potentially sensitive uploads to stick around longer than desired on the temporary path.